src/Symfony/Component/Form/FormRenderer.php

```diff
@@ -316,6 +316,6 @@ public function searchAndRenderBlock(FormView $view, $blockNameSuffix, array $va
      */
     public function humanize($text)
     {
-        return ucfirst(trim(strtolower(preg_replace(array('/([A-Z])/', '/[_\s]+/'), array('_$1', ' '), $text))));
+        return ucfirst(strtolower(trim(preg_replace(array('/([A-Z])/', '/[_\s]+/'), array('_$1', ' '), $text))));
```

Comment:
Potentially un-needed work: 
- was: strtolower (including trimmable spaces) -> trim
- become: trim (reduce work for strtolower) -> strtolower